apps/pom.xml

```diff
@@ -93,6 +93,8 @@
 
 		<metrics.version>4.1.2</metrics.version>
 		<hibernate.version>5.6.1.Final</hibernate.version>
+		<junit.version>5.8.2</junit.version>
+		<junit.vintage.version>5.8.2</junit.vintage.version>
```

Comment:
assuming these are kept in lock step -- do we need two properties that could inadvertently diverge?